keps/sig-cli/kubectl-socket-support.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Socket Support for Kubectl
+authors:
+  - "@choo-stripe"
+  - "@dixudx"
+  - "@answer1991"
+owning-sig: sig-cli
```

Comment:
because this is implemented in the client library, and merely inherited by kubectl, I think this might actually be sig-api-machinery, with sig-cli as a participating sig